spec/features/authentication/lock_user_account_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'support/unlock_account_use_case_spy'
+require 'support/unlock_account_use_case'
+
+describe "Locking a users account after ten failed login attempts" do
```

Comment:
I would tweak some of these context blocks / it descriptions - might be easier to talk through it together?